[RHDX-412] Rework cta styles and add light-blue theme

alistairmc opened the pull request:
### JIRA Issue Link
https://projects.engineering.redhat.com/browse/RHDX-412

### Verification Process
check cat components on 
https://docs.google.com/spreadsheets/d/1cD6W-nwd4wqyhMclD_kqJffnmOvjM_RNA8sT_C-UXoc/edit#gid=0

Test any CTA_assembly you should have the option for dark theme and light-blue theme, also left centre and right align.

Now if you have a background Image you can select dark to have white text and leave without a theme class to maintain dark text if the background image is light.

will require content changes need to discuss with UX , plan to make changes (Alistair) 40 components.

Tc confirm Alistair will make the changes to the live content after release.

staceymosier commented:
@alistairmc  @sjcox-rh 
These pages have call_to_actions with dark background images with black text.
Can we update them in this env so that they appear how you expect them to?

- https://developer-preview-3542.ext.us-west.dc.preprod.paas.redhat.com/products/rhel/ubi/?type=assembly_page
- https://developer-preview-3542.ext.us-west.dc.preprod.paas.redhat.com/topics/microservices
- https://developer-preview-3542.ext.us-west.dc.preprod.paas.redhat.com/training
- https://developer-preview-3542.ext.us-west.dc.preprod.paas.redhat.com/products/

staceymosier commented:
@alistairmc 
Can you include a temporary patch that any CTA with a background image or within a Two-Col assembly will by default have white text, until the content has been assigned the correct visual styles?

Please ping me in the morning if you need more context for this request.

alistairmc commented:
![Screenshot 2020-04-08 at 09 17 12](https://user-images.githubusercontent.com/900631/78761693-daad8f80-797a-11ea-8664-27f1fd30604f.png)
![Screenshot 2020-04-08 at 09 16 52](https://user-images.githubusercontent.com/900631/78761706-de411680-797a-11ea-9c1f-3922f34a9f04.png)

Attached is  a list of all cta assemblies would take about an hour to fix in live and most look fine anyway until updated.

@staceymosier I would prefer not to add a patch since I would all components to behave consistently. @sjcox-rh what is the UX view on this?

alistairmc commented:
@staceymosier I have updated all the content in this PR

alistairmc commented:
@staceymosier I have updated the cts's in live and added dark to the components with backgrounds. Once this is merged and deployed I can make addition updates to live to add the centre class and light-blue theme where needed. @sjcox-rh can you give this your ok please
